Core/Core/AppEnvironment/AppleManagedAppConfiguration.swift

```diff
@@ -0,0 +1,86 @@
+//
+// Copyright (C) 2019-present Instructure, Inc.
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, version 3 of the License.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+//
+
+import Foundation
+
+/// Manages Apple MDM Configuration
+///
+/// You can test this locally with command line arguments
+// -com.apple.configuration.managed '<dict><key>enableDemo</key><true/><key>username</key><string>username</string><key>password</key><string>password</string></dict>'
+/// `username` and `password` values should match what's set in App Store Connect.
+public class AppleManagedAppConfiguration {
```

Comment:
This seems very specific to AppStore review process, but doesn't really handle how schools could use MDM.